ansible.cfg

```diff
@@ -7,6 +7,8 @@ host_key_checking=False
 gathering = smart
 fact_caching = jsonfile
 fact_caching_connection = /tmp
+# Once created, fact cache is never force updated. Enable this if you're getting stale data in your runs (e.g. stale `etcd_access_addresses`)
+# fact_caching_timeout = 7200 # 2 hours
```

Comment:
Can we just enable it by default? It's not a bad choice

Reply:
To be honest, i was struggling with these for a few hours - being new to Ansible and all... 
Only reason i left it commented out was for backwards-compatibility

Reply:
Yeah, agree, let's enable it by default 👍

Reply:
done ;)